packages/react/src/account/useCreateAccountWithUsername.ts

```diff
@@ -0,0 +1,49 @@
+import type {
+  OperationHandler,
+  SigningError,
+  TransactionIndexingError,
+  UnauthenticatedError,
+  UnexpectedError,
+  ValidationError,
+} from '@lens-protocol/client';
+import {
+  createAccountWithUsername,
+  fetchAccount,
+} from '@lens-protocol/client/actions';
+import type {
+  Account,
+  CreateAccountWithUsernameRequest,
+} from '@lens-protocol/graphql';
+import { expectTypename, nonNullable } from '@lens-protocol/types';
+
+import { type UseAsyncTask, useAuthenticatedAsyncTask } from '../helpers';
+
+export type UseCreateAccountWithUsernameArgs = {
+  handler: OperationHandler;
+};
+
+/**
+ * Creates an account on Lens.
```

Comment:
In general, let's always use Account (upper cased A) when we refer to the Lens Account smart contract (like in docs).